deploy/helm/sumologic/values.yaml

```diff
@@ -630,15 +636,13 @@ otelcol:
       batch:
         # Number of spans after which a batch will be sent regardless of time
         send_batch_size: 256
-        # Number of tickers that loop over batch buckets
-        num_tickers: 10
         # Time duration after which a batch will be sent regardless of size
         timeout: 5s
     extensions:
       health_check: {}
     exporters:
       zipkin:
-        url: "exporters.zipkin.url_replace"
+        url: {{ .Values.sumologic.traces.endpoint || quote }}
```

Comment:
helm templating doesn't work in `values.yaml`
Thats why there is string which is replaced later in `traces.conf` file

Reply:
also `||` -> `|`